app/controllers/bands_controller.rb

```diff
@@ -1,5 +1,34 @@
 class BandsController < ApplicationController
+  before_action :set_band, only: :show
+
+  def index
+    @bands = Band.all
+  end
   def show
+  end
+
+  def new
+    @band = Band.new
+  end
+
+  def create
+    @band = Band.new(band_params)
+    # TODO Replace User.first with current_user when available
+    @band.user = User.first
```

Comment:
```suggestion
    # TODO Replace User.first with current_user when available
    @band.user = current_user
```
current_user is available